lib/PostHog/Client.php

```diff
@@ -22,7 +21,7 @@ class PostHog_Client {
    */
   public function __construct($apiKey, $options = array()) {
     $consumers = array(
-      "socket"     => "PostHog_Consumer_Socket",
+      "socket"     => "Socket",
```

Comment:
How come this is renamed but the other ones aren't?

Reply:
The way we are working with the classes here is not-optimal, I have to rewrite most of it but I ran "out of budget" for this PR.